Temporarily revert https://github.com/hrydgard/ppsspp/commit/94bab4506f2e
until files/patch-ffmpeg5 is adapted

Core/HW/MediaEngine.cpp:427:38: error: no member named 'codec' in 'AVStream'
                        m_codecsToClose.push_back(stream->codec);
                                                  ~~~~~~  ^

--- Core/HW/MediaEngine.cpp.orig	2023-04-30 11:42:05 UTC
+++ Core/HW/MediaEngine.cpp
@@ -351,11 +351,6 @@ void MediaEngine::closeContext()
 #endif
 	}
 	m_pCodecCtxs.clear();
-	// These are streams allocated from avformat_new_stream.
-	for (auto it : m_codecsToClose) {
-		avcodec_close(it);
-	}
-	m_codecsToClose.clear();
 	if (m_pFormatCtx)
 		avformat_close_input(&m_pFormatCtx);
 	sws_freeContext(m_sws_ctx);
@@ -415,8 +410,6 @@ bool MediaEngine::addVideoStream(int streamNum, int streamId) {
 			if (streamNum >= m_expectedVideoStreams) {
 				++m_expectedVideoStreams;
 			}
-
-			m_codecsToClose.push_back(stream->codec);
 			return true;
 		}
 	}
--- Core/HW/MediaEngine.h.orig	2023-04-30 11:42:05 UTC
+++ Core/HW/MediaEngine.h
@@ -126,7 +126,6 @@ class MediaEngine
 #ifdef USE_FFMPEG
 	// Video ffmpeg context - not used for audio
 	AVFormatContext *m_pFormatCtx = nullptr;
-	std::vector<AVCodecContext *> m_codecsToClose;
 	AVIOContext *m_pIOContext = nullptr;
 	SwsContext *m_sws_ctx = nullptr;
 #endif
